WcaOnRails/app/assets/javascripts/persons.js

```diff
@@ -0,0 +1,34 @@
+// Add params from the search fields to the bootstrap-table for on Ajax request.
+function queryParams(params) {
```

Comment:
I see. How about doing a quick scoping like this?

```javascript
var personsTableAjax = {
  queryParams: function(params) { ... },
  doAjax: function(options) { ... },
};
```

(Feel free to rename things as appropriate.) This should allow you to access these functions as `personsTableAjax.queryParams`.